lldb/source/Plugins/ObjectFile/ELF/ObjectFileELF.cpp

```diff
@@ -935,6 +935,16 @@ lldb_private::Address ObjectFileELF::GetEntryPointAddress() {
 }
 
 Address ObjectFileELF::GetBaseAddress() {
+  if (GetType() == ObjectFile::eTypeObjectFile) {
+    for (SectionHeaderCollIter I = std::next(m_section_headers.begin());
+         I != m_section_headers.end(); ++I) {
```

Comment:
For my own understanding, why do we start at the 2nd section header?

Reply:
> For my own understanding, why do we start at the 2nd section header?

The first one is all zeroes in every ELF file